app-sso/concepts/levels-of-consumption.hbs.md

```diff
@@ -0,0 +1,287 @@
+# The three levels of AppSSO consumption
+
+This topic describes the different levels of consuming AppSSO services and
+explains when and why you might choose to use one level over the other.
+
+Generally, the recommendation is to consume AppSSO through `ClassClaim` but
+there can be situations where the lower level `WorkloadRegistration` or
+`ClientRegistration` are a better fit.
+
+At its core consumption of AppSSO is about obtaining client credentials for an
+authorisation server and loading them into a running workload. In particular, it
+breaks down into the following steps:
+
+1. Define your environment-independent OAuth2 client configurations, e.g. client
+   authentication method, scopes, ...
+
+1. Define your OAuth2 client's redirect URIs
+
+1. Specify the authorisation server that you want credentials for
+
+1. Create a resource that expresses your configuration
+
+1. Mount the client credentials into a workload
+
+Each of the following levels gradually takes away some of these steps
+by distributing them across APIs, so that eventually, each persona
+is only concerned with what's in their purview:
+
+* _Platform operators_ manage the TAP (and AppSSO) installation
```

Comment:
[nit]: the other 2 are plural, so maybe this can be `installations`

Reply:
perfect! ty